assets/src/components/activities/check_all_that_apply/CheckAllThatApplyDelivery.tsx

```diff
@@ -105,7 +105,7 @@ export const CheckAllThatApplyComponent: React.FC = () => {
 // Defines the web component, a simple wrapper over our React component above
 export class CheckAllThatApplyDelivery extends DeliveryElement<CATASchema> {
   render(mountPoint: HTMLDivElement, props: DeliveryElementProps<CATASchema>) {
-    const store = configureStore({}, activityDeliverySlice.reducer);
+    const store = configureStore({}, activityDeliverySlice.reducer, { name: 'CATADelivery' });
```

Comment:
At one point, I had a theory about multiple redux stores being the cause of this bug, and the redux-dev tools were very difficult to use because they weren't labeled. It didn't end up being the case, but I did label these to make it easier for future debugging.